### DISCUSSION TITLE
How to establish SQL connection from MacBook to Windows server

### DISCUSSION THREAD

**Discussion by Susja** [community/community#67907] [New to GitHub]
Hello

I am using ADS on Windows PC
![Screen Shot 2023-09-21 at 6 28 11 PM](https://github.com/community/community/assets/26395390/2e4d1499-3600-49d3-8916-c20034ec6037)

I have created a few SQL Connections from my Windows PC to SQL servers

Now I have to use MacBook and I still want to connect to the same Windows SQL servers I connected before from Windows PC

I installed Visual Studio Code on MacBook. It works fine .. but I failed to establish SQL connection.

I could ping SQL server but not connect. I tried Integrated and SQL authentication but it did not work.

I have mssql error: Unable to connect

Any suggestions how to establish connection in my case?

Thanks

**Comment by queenofcorgis** [comment]
Hi @Susja ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Azure Data Studio like their community: https://www.microsoft.com/en-us/sql-server/community?activetab=pivot:sqlservertab. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
